src/test/java/uk/gov/digital/ho/pttg/api/HmrcResourceContractTest.java

```diff
@@ -34,6 +36,9 @@
     @Autowired
     private MockMvc mockMvc;
 
+    @Rule
+    public OutputCapture outputCapture = new OutputCapture();
```

Comment:
This is such a better pattern than redirecting System.out to a buffer which is how I've always done it - I hope I remember this when I next need to assert on output!